### DISCUSSION TITLE
Cannot Install CLI on M1 Chip Macbook

### DISCUSSION THREAD

**Discussion by bimberman** [deta/cloud-docs#553] [Help]
<img width="1242" alt="Screen Shot 2022-10-25 at 9 27 24 AM" src="https://user-images.githubusercontent.com/2808668/197831542-ee40b945-7d49-46d3-8271-3dcce8add018.png">

After successfully installing the CLI I am not able to login.
I tried editing the PATH in the .zshrc file but it did not help.
Thank you in advance for your help.

**Comment by AyushSehrawat** [comment]
Try using full path once like
```sh
/Users/benimberman/.deta/bin/deta --help
```

**Comment by abdelhai** [comment]
@bimberman restart your terminal
